middleman-core/lib/middleman-core/renderers/haml.rb

```diff
@@ -52,6 +52,10 @@ def self.render_with_options(text, compiler_options)
               modified_options = options.dup
               modified_options[:context] = compiler_options[:context]
 
+              renderer_symbol = self.to_s.split(":").last.downcase.to_sym
+              renderer_options = modified_options[:context].app.config[renderer_symbol]
+              modified_options = modified_options.merge(renderer_options || {})
```

Comment:
Looks like this won't work in the context of a Haml filter because the `context` is `nil`. If anyone can think of a way to retrieve the `config` some other way, I'll take the hint. 😄